### ISSUE TITLE
Pocket Beagle: USB tethering does not work in V5.3

### ISSUE THREAD

**Issue by Old-Salt** [FalconChristmas/fpp#1143]
**Describe the bug and Steps to reproduce**: 

Pocket Beagle - no cape / Windows 7
- Tried flashing a new SD Card with FPP-v5.3-BBB.img.zip (SD Card Formatter/Balena)
Unable to connect with USB tethering (http://192.168.7.2)
- Tried flashing a known good SD Card with FPP-v5.3-BBB.img.zip (SD Card Formatter/Balena)
Unable to connect with USB tethering (http://192.168.7.2)
- Tried flashing a known good SD Card with FPP-v5.3-BBB.img.zip (Raspberry Pi Imager)
Unable to connect with USB tethering (http://192.168.7.2)

Flashed FPP 4.6.1 known good SD Card - USB tethering works
Expanded the file system
Uploaded BBB-5.3.fppos
Upgraded the OS using FPP
Tethering no longer works

**Expected behavior**: 
Working USB tether

**Additional context**: 

**Additional Attachments**

**Comment by bcarriveau** [comment]
Salt is on windows 7, maybe there is an additional driver that can be added to fpp 5.3?


Myself, i am on windows 10 21h1 and Tether to 5.3 on a PB works with 192.168.7.2

**Comment by Old-Salt** [comment]
Did some more checking with earlier versions of FPP:

Flash FPP-v5.1.1-BBB.img.zip: USB Device not recognized / no USB tethering (192.168.7.2)
Flash FPP-v5.0-BBB.img.zip: USB Device not recognized / no USB tethering (192.168.7.2)
Flash FPP-v4.6.1-BBB.img.zip: USB Device recognized / USB tethering active (192.168.7.2)

Done on a Windows 7 system. Using a USB 2 or 3 port makes no difference.  Flash procedure was format with SD Card Formatter followed by write using Balena Etcher.
USB device recognized indicated by a system beep.
Note the four system lights flash differently during the Pocket Beagle boot sequence between the working FPP 4.6.1 and versions 5.0 and up.

**Comment by bloojoop** [comment]
I was also helping to test this.  I'm on windows 10 20H2.  When I have OS5.0 (any version of fpp) it boot when attached to USB without issue.  I first flashed with 5.2 to get os 5.0 and usb tethering worked as expected.  I reattached to the cape and upgraded fpp to 5.3.1 and usb tethering still worked as expected.  I then upgrade to os 5.3 and fpp 5.31 by using the fppos method and usb tethering does not work.   I observed when on OS 5.0, the pocket beagle boot lights would light up and flash signifying the boot up process.  When on OS 5.3, I get the power light only and never get the other system lights to flash at all.  It appears as if it is not booting up when using usb tethering and OS 5.3.

**Comment by bloojoop** [comment]
@dkulp ,should it work with win 10 20h2 ?  I have the same issue with fpp OS 5.3, but not with OS 5.0 ?  I don't rely on usb tethering but just curious as I seem to have the same issue.

**Comment by Old-Salt** [comment]
I did some more testing following receipt of my order from Dan...

The short result is that **_USB tethering works with the Pocket Beagle and Windows 7.   The problem was due to improper seating of the SD card._**  

Note that when first reporting this as a bug I had tried three different SD Cards and made more than twenty attempts to use USB tethering.  All three SD cards have since been used successfully on a Pocket Beagle without a Cape for USB tethering.

**How Found:**
I removed the Beaglebone Black from the K32A-B I received with my order and installed one of the SD cards with the image of FPP 5.0.  When I connected it to my Windows 7 computer it tried to install drivers. Two of the four failed to install, but RNDIS 4.0 was successfully installed.  Thinking that this might be a work-around, I tried a Pocket Beagle mounted on a K4 with the OLED display, and no USB WiFi adapter.  On the display I noted that the Pocket Beagle did not boot properly.  It remained blank.  I had to reseat the same SD card several  times before I had a successful boot.  I removed the Pocket Beagle from the K4 and it booted successfully.

At this point, I could not determine whether the problem originally caused by the missing RNDIS driver or the seating of the SD card.  To make the determination, I brought an identical Windows 7 computer that had never been connected to a Beaglebone Black from storage. The bare Pocket Beagle booted successfully, with USB tethering, and with the installation of the RNDIS driver.  I re-installed the Pocket Beagle on the K4 with the OLED display.  The same SD card was removed and re-installed several times, with a failed boot more often than not.  This showed that the original problem was due to improperly seated SD cards.

**Recommendation:**  Watch the lights on a bare Pocket Beagle on a Pocket Beagle if you are familiar with their action during a successful boot.  If you are not, install the Pocket Beagle on a cape with an OLED display, and watch for the logo and boot messages to appear.  If they don't, or the lights on the Pocket Beagle do no react as during a normal boot, re-seat the SD card.  It may take several attempts.
